_assets/scripts/push_benchmark.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REPO_URL="git@github.com:status-im/status-go-benchmarks.git"
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+source "${GIT_ROOT}/_assets/scripts/colors.sh"
+
+echo -e "${GRN}Pushing benchmark results${RST}"
+
+cd "${GIT_ROOT}"
+
+git clone "${REPO_URL}" benchmarks-repo
+cd benchmarks-repo
+
+timestamp=$(date -u '+%Y%m%dT%H%M%S')
+commit_sha=$(git rev-parse --short HEAD)
```

Comment:
Aren't you using the previous commit of benchmark repo instead of the `status-go` repo for the folder name?